hydra-node/src/Hydra/Chain/Direct/Wallet.hs

```diff
@@ -362,7 +362,7 @@ estimateScriptsCost pparams systemStart epochInfo utxo tx = do
     Left translationError ->
       Left $ ErrTranslationError translationError
     Right units ->
-      Map.traverseWithKey (\ptr -> left $ ErrScriptExecutionFailed . (ptr,)) units
+      traverse (left ErrScriptExecutionFailed) units
```

Comment:
Hahah.. @ffakenz, I happened to remove it here as well :sweat_smile:. I should re-add the redeemer pointer.